finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,7 +273,7 @@ def load_f460A_row(row, agency, verbosity=1):  # noqa
     beneficiary.save()
 
     # Now we have all the parts, save!
-    money = models.IndependentMoney(  # or calculated_Amount
+    money, _ = models.IndependentMoney.objects.get_or_create(  # or calculated_Amount
```

Comment:
This should simply `get_or_create` with the two unique items, then set the other items. Otherwise, if there's a manual annotation, this won't match and you'll get an error.